sunkit_image/tests/test_enhance.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pytest
+
+import sunpy.map
+import sunpy.data.sample
+from sunpy.tests.helpers import figure_test
+
+import sunkit_image.enhance as enhance
+
+
+@pytest.fixture
+@pytest.mark.remote_data
+def smap():
+    return sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
```

Comment:
Switch to the sunpy test data for the figures. It would allow us to drop the remote_data flag.